lib/feedsme/index.js

```diff
@@ -28,10 +28,17 @@ function Feedsme({ agents, models, config, log }) {
   var proto = url.parse(uri).protocol;
 
   this.wrhs = new Warehouse(config.get('warehouse'));
-  this.carpenter = new Carpenter({
-    uri: uri,
-    agent: agents[proto]
-  });
+  this.carpenter = {
+    v1: new Carpenter({
```

Comment:
Does feedsme still need a v1 client?

Reply:
shouldn't this just be plumbing the `promote` bit through everything? And the feedsme v1 endpoint just always sets `promote=true` but always still talks to the v2 carpetnerd endpoint?